src/eHealth.locations/services/selected-location-factory.js

```diff
@@ -246,8 +246,19 @@ angular.module('eHealth.locations.services')
             }
           });
           return angular.copy(adminDivisions);
+        },
+        clear: function () {
+          levels.forEach(function (level) { delete level.selected });
         }
       };
+
+      Object.defineProperty(location, 'innermost', {
```

Comment:
Basically `location.innermost.id` vs. `location.innermost().id`. I like the former better.